ou_dedetai/control.py

```diff
@@ -38,6 +38,7 @@ def restore(app=None):
     backup_and_restore(mode='restore', app=app)
 
 
+# FIXME: consider moving this into it's own file/module.
```

Comment:
I think utils is the place this probably should be in the current file structure.

Its existence here is a holdover from the bash script.

This functionality I think could easily be in its own module, or perhaps we could refactor things to have our various copy functions or otherwise all in a files.py or something similar.

Reply:
backup.py makes sense to me, there's a lot of logic here